### ISSUE TITLE
[css-content-3][css-values-5] Define impact of invalid namespaces prefixes within attr()

### ISSUE THREAD

**Issue by fantasai** [w3c/csswg-drafts#6813]
```
::before { content: attr(x|y) "pass or fail?" }
```

It's not clearly defined what should happen here. But Firefox (which supports namespaces within attr(); WebKit and Blink do not) treats it as not invalidating the entire rule (whereas invalid namespaces within selectors *are* treated as invalidating the rule). Seems reasonable to define it as "can't find the attribute" rather than "invalidates declaration", so probably we should just spec what Firefox does here.

**Comment by tabatkins** [comment]
Yes, the Selectors behavior of making the selector itself invalid is *nonsensical* and we absolutely shouldn't spread it, in any variation, to anything else.

An invalid/missing namespace prefix just means we don't know what the attribute is and cause a substitution failure, definitely.
